cluster_test.go

```diff
@@ -154,9 +154,8 @@ func testingCluster(t *testing.T) (*Cluster, *mockAPI, *mockConnector, state.Sta
 
 	raftcon, _ := raft.NewConsensus(host, consensusCfg, st, false)
 
-	bmonCfg.CheckInterval = 2 * time.Second
 	psmonCfg.CheckInterval = 2 * time.Second
-	mon := makeMonitor(t, host, bmonCfg, psmonCfg)
+	mon := makeMonitor(t, host, psmonCfg)
```

Comment:
`makeMonitor` function is not necesary anymore, instantiate the monitor directly